src/modules/CreateChannel/context/CreateChannelProvider.tsx

```diff
@@ -26,71 +26,101 @@ type OverrideInviteUserType = {
 
 export interface CreateChannelProviderProps {
   children?: React.ReactElement;
-  onCreateChannel(channel: GroupChannel): void;
+  userListQuery?(): UserListQuery;
+
+  /**
+   * Overrides the action of the channel creation button.
+   * */
+  onCreateChannelClick?(params: OverrideInviteUserType): void;
```

Comment:
I think we should rename this to `onClickCreateChannel`
What do you think about it?

Reply:
cc @bang9 @AhyoungRyu @liamcho